Framework/Core/include/Framework/DataAllocator.h

```diff
@@ -313,20 +314,51 @@ class DataAllocator
                   "pointer to data type not supported by API. Please pass object by reference");
   }
 
+  template <typename T>
+  auto make(OutputRef const& ref)
+  {
+    return make<T>(getOutputByBind(ref));
+  }
+
+  template <typename T>
+  auto make(OutputRef const& ref, size_t n)
```

Comment:
is that already part of the variable argument list below? Or maybe the specific substitution  is necessary because of the return type, I remember that I had some problem in such a situation.